tests/integration/README.md

```diff
@@ -584,15 +584,9 @@ The test framework supports the following command-line flags:
   -istio.test.kube.systemNamespace string
         The namespace where the Istio components reside in a typical deployment. (default "istio-system")
 
-  -istio.test.kube.helm.chartDir string
```

Comment:
This is still used by the default values (values-integ.yaml). We should probably just make the defaults somewhere else and then remove it though, rather than keep this around.

If you do remove from the docs though we should also remove the flag in code

Reply:
Tbh I think we don't even need those defaults; I think every test should clearly state the setup it requires, without some magic test defaults. Default should be user-default IMO

Reply:
I think a lot of it makes sense. We don't want pods requesting high CPU since it won't run in small clusters properly. Turning on access logs by default also makes sense to help debugging. A lot of it or maybe all of it can probably go away though

Reply:
OK, the resource limits make sense. Will add the defaults back, but probably move them to a different location (ie somewhere in tests/)

Reply:
Actually, re-introducing the defaults will be a larger change, because in combination with `ControlPlaneValues`, you need to merge things. I will do that in a follow-up PR

Reply:
You don't need custom logic to merge it if you have a IOP file since istioctl takes multiple `-f` arguments now by the way

Reply:
Thanks, didn't know about the `-f` flag chaing. I'm now using that in the follow up PR: https://github.com/istio/istio/pull/21144